limitador/src/prometheus_metrics.rs

```diff
@@ -3,6 +3,26 @@ use prometheus::{Encoder, IntCounterVec, IntGauge, Opts, Registry, TextEncoder};
 
 const NAMESPACE_LABEL: &str = "limitador_namespace";
 
+struct Metric {
+    name: String,
+    description: String,
```

Comment:
these could very easily be &'a str since all contents are static strings

Reply:
Right, but there are only 3 instances of this, so I don't think it's worth it.